content/blog/2018/07/devops-world-jenkins-world-2018-agenda-is-live.adoc

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: "DevOps World-Jenkins World 2018 Agenda is Live"
+tags:
+- event
+- jenkins world
```

Comment:
Tags with spaces look like a risky thing. I do not know how the engine handles them